documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+= Basic Maven Configuration
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+
+In the context of application development, Maven allows you to extend your applications by referencing and utilizing resources from local and remote repositories. This article serves as an introduction to using Maven with the IDE and contains instructions for the following tasks:
+
+. <<New_Project,Create a New Maven Project>>
+. <<New_Module,Create a New Maven Module>>
+. <<Maven_Support,Add Maven Support to an Existing Non-Maven Project>>
+
+Maven projects serve as parents for Maven modules. For each Maven module attached to a Maven project, a section is added to the project’s pom.xml file. Each command run by the Maven project is run against each of the included modules first.
+
+[[New_Project]]
+== 1. Create a New Maven Project
+The first step when using Maven with an IDE is to create a Maven project that can contain multiple independent Maven modules. Follow the instructions to create a Maven project:
+
+. Create a new project:
+.. In the workspace, navigate to menu:File[New > Other].
+.. Type `maven` in the `Filter` field and select `Maven Project`.
+.. Click `Next` to continue.
+. In the first step of the `New Maven Project` wizard, enter the initial project details:
+.. Select the `Use default Workspace location` check box to use the default workspace. If required, specify a non-default location by deselecting the check box and using the `Browse` button to specify the desired target directory.
```

Comment:
Done. New text: "Ensure that the Use default Workspace location check box is not selected and specify a non-default location for your workspace files using the Browse button. Using a non-default workspace location is recommended because this allows other tools to access the workspace location easily."